### PR TITLE
img tag refactor in CONTRIBUTING.md

### LINKED ISSUE #5620: img tag refactor in CONTRIBUTING.md Section 4.1 making
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We want to change an img HTML tag ending with a slash () to an img tag without an ending slash () so that the codebase is consistent with how we use img HTML tags.

### Action Items
- [x] In your local IDE, navigate to `CONTRIBUTING.md`, Section 4.1
- [x] Replace
```
<img src="https://user-images.githubusercontent.com/21162229/137784250-96281332-9231-4c5d-aba2-0d4c36521b82.jpg" alt="Making a comment visual example" />
```
with
```
<img src="https://user-images.githubusercontent.com/21162229/137784250-96281332-9231-4c5d-aba2-0d4c36521b82.jpg" alt="Making a comment visual example">
```
- [x] Changes to CONTRIBUTING.md cannot be tested locally, rather they must be tested after pushing the issue branch to your fork of the repository.  Push your issue branch in the usual manner, but before creating the Pull Request, check your updates using this test URL.  Also store the test URL for use in a later step:
```
https://github.com/[REPLACE WITH GITHUB HANDLE]/website/blob/[REPLACE WITH NAME OF ISSUE BRANCH]/CONTRIBUTING.md
```
   (for example: https://github.com/bonniewolfe/website/blob/issue-branch-1234/CONTRIBUTING.md)
- [x] Create a pull request with your changes.  In the Pull Request, after the "Why did you make the changes" section, add this line to help reviewers, replacing the text in brackets (and the brackets) with the test URL from the previous Action Item.
```
For Reviewers: Do not review changes locally, rather, review changes at [REPLACE WITH TEST URL]
```

### Resources/Instructions
[CONTRIBUTING.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
[Getting started with writing and formatting on GitHub](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github)
- This issue is tracked in [HfLA-Website: img Tag Audit spreadsheet's img with Ending Slash tab](https://docs.google.com/spreadsheets/d/1cq8iLOYQkAbho2GsRAxVJYgjxDeJ9KLdx2NuL4G6vu4/edit#gid=0), and #5040

### PR DESCRIPTION
**Pull request by Chrisklangley**
Fixes #5620 

### What changes did you make?
  - on line 990 of the contributing.md file I replaced existing image tag with new one Via issue details 
  
  

### Why did you make the changes (we will use this info to test)?
For Reviewers: Do not review changes locally, rather, review changes at https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Making a comment visual example

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
Making a comment visual example

</details>

### CODE REVIEW

**Comment by jaasonw** [comment]
eta: 10/20 eod
availability: 10/20 all day

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -987,7 +987,7 @@ The best way to ask for help and provide as much information to the team is to d
 
 <details>
   <summary><strong>Click here</strong> for visual comment example</summary>
-  <img src="https://user-images.githubusercontent.com/21162229/137784250-96281332-9231-4c5d-aba2-0d4c36521b82.jpg" alt="Making a comment visual example" />
```
Comment (jaasonw) [inline CONTRIBUTING.md:990]: you should re add the space here to keep the indentation consistent

File: package-lock.json [hunk 2]
```diff
@@ -1,3 +1,6 @@
 {
-  "lockfileVersion": 1
```
Comment (jaasonw) [inline package-lock.json:2]: you might have accidentally committed this but this package-lock.json file isnt relevant to the pr

**Review by jaasonw — CHANGES_REQUESTED** [review]
visually, it looks good to me but I added some suggestions and I also suggest
a. making the title more descriptive 
b. small nitpick but you didn't have to screenshot the code change
image

File: CONTRIBUTING.md (hunk 1 above)
Comment (Chrisklangley) [reply CONTRIBUTING.md:990]: DO you mean a space between the details tag? Can you please explain what you mean? Thanks @jaasonw

Comment (jaasonw) [reply CONTRIBUTING.md:990]: referring to the space that was deleted here
image

**Review by jaasonw — CHANGES_REQUESTED** [review]
your package-lock.json changes are still committed and it appears you undid your original changes
(see: https://github.com/hackforla/website/pull/5745/files)

**Pull request closed without merging**